File: Vjezbe_5/Zadatak1.py
import harmonic_oscillator as ho                #type: ignore

tijelo = ho.HarmonicOscillator()

mass = 4.5
constant = 140
velocity = 5
position = 10
time = 3.1
dt = 0.001

tijelo.set_initial_conditions(mass, constant, velocity, position)
tijelo.plot_trajectory(time, dt)

File: Domaci_3/Zadatak1.cpp
#include <iostream>
#include <HarmonicOscillator.h>



int main(){

    double time = 4.0;

    HarmonicOscillator oscilator(-10, 0, 2.5, 150, time);
    oscilator.write_file();

    return 0;
}